plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesPluginImpl.kt

```diff
@@ -988,13 +992,14 @@ class GesturesPluginImpl : GesturesPlugin, GesturesSettingsBase {
       duration = animationTime
     }
 
-    zoomAnimator.addListener(object : AnimatorListenerAdapter() {
```

Comment:
any reason removing the zoomAnimator update listener?

Reply:
yes, we actually create zoom + anchor animator and run them as a set.
so it's more logical to reset anchor when anchor animator ends (although it's technically same moment when zoom animator ends).